homeassistant/components/mysensors.py

```diff
@@ -337,7 +338,8 @@ def update(self):
             _LOGGER.debug(
                 "%s: value_type %s, value = %s", self._name, value_type, value)
             if value_type in (set_req.V_ARMED, set_req.V_LIGHT,
-                              set_req.V_LOCK_STATUS, set_req.V_TRIPPED):
+                              set_req.V_LOCK_STATUS, set_req.V_TRIPPED,
+                              set_req.V_DIMMER):
```

Comment:
This should not be here, but in a separate check. The value for V_DIMMER should just be forced to an int, not changed to a home assistant state. Make it an `elif`.